test/src/test/java/org/corfudb/runtime/view/SnowflakeGuidGeneratorTest.java

```diff
@@ -0,0 +1,49 @@
+package org.corfudb.runtime.view;
+
+import lombok.Getter;
+import org.corfudb.runtime.CorfuRuntime;
+import org.junit.Test;
+
+import java.util.HashSet;
+import java.util.Random;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Created by Sundar Sridharan on May 23, 2019.
+ */
+public class SnowflakeGuidGeneratorTest extends AbstractViewTest {
+
+    @Test
+    public void areUniqueAndOrderedLong() {
+        final int iterations = 1000;
+        CorfuRuntime r = getDefaultRuntime();
```

Comment:
Actually you can create your own instance of CorfuRuntime. Then you don't need to extend AbstractViewTest